backend/lambda/tokenize/lambda_function.py

```diff
@@ -0,0 +1,49 @@
+"""
+This is snapshot of what is currently on AWS Lambda function, named 'sw-test'
+Last updated on 3/16/2022 @ 16:38 PST
+
+This live version on AWS can be updated anytime, and this snapshot WILL NOT always be in-sync with the live version.
+"""
+import json
+import nltk
+from tokenizer import Tokenizer
+
+
+def lambda_handler(event, context):
+    result = {}
+    invalidRequest = False
+
+    try:
+        text = event["text"]
+        op = event["path"]
+    except:
```

Comment:
Using a general exception statement is poor practice. An appropriate error, e.g. KeyError.

Reply:
Good point, I will specify the exception type